operator/pkg/operator.go

```diff
@@ -331,7 +334,25 @@ func (o *Operator) ProcessNewBatchLogV3(newBatchLog *servicemanager.ContractAlig
 	return nil
 }
 
+func (o *Operator) IsVerifierValid(blacklisted_verifiers_bitmap *big.Int, verifierId common.ProvingSystemId) bool {
+	verifierIdInt := uint8(verifierId)
+	bit := uint8(blacklisted_verifiers_bitmap.Uint64()) & (1 << verifierIdInt)
```

Comment:
Are we sure we won't have more than eight different verifiers?

Reply:
The bitmap is implemented as a `u256` so it makes sense to enforce `u8`s as indexes as we save ourselves from checking if the provided number is between the boundary of the bitmap (0-255). Though, why are we casting `blacklisted_verifiers_bitmap` to an `uint64` instead of `uint256`?

Reply:
We're casting the bitmap to `uint8` before applying the mask, which is a single bit shifted by the id number, so that limits us to 8 different verifiers.

Reply:
You're right, thanks for pointing that out. It works now because we have fewer than 8 verifiers, but it's cutting it close. I changed it to use `uint64` directly, which should be fine since we likely won't need more than 64.

Reply:
> 64 verifiers ought to be enough for anybody
- Not Bill Gates